app/src/main/res/values/strings.xml

```diff
@@ -599,4 +598,10 @@ Upload your first media by tapping on the add button.</string>
   <string name="wallpaper_set_unsuccessfully">Something went wrong. Could not set the wallpaper</string>
   <string name="setting_wallpaper_dialog_title">Set as Wallpaper</string>
   <string name="setting_wallpaper_dialog_message">Setting Wallpaper. Please wait…</string>
+  <string name="theme_default_name">Default</string>
+  <string name="theme_dark_name">Dark</string>
+  <string name="theme_light_name">Light</string>
+  <string name="theme_default_value">0</string>
```

Comment:
You actually don't want to specify these as string resources, now they will get translated and that will lead us to back to an original problem

Reply:
Done!